cmd/minikube/cmd/config/profile.go

```diff
@@ -47,6 +47,12 @@ var ProfileCmd = &cobra.Command{
 		profile := args[0]
 		if profile == "default" {
 			profile = "minikube"
+		} else {
+			// not validating when it is default profile
+			errProfile, ok := ValidateProfile(profile)
+			if !ok && errProfile != nil {
+				out.FailureT(errProfile.Msg)
```

Comment:
this PR looks good ! but we need to add a message to the end user, how to create a profile !

this PR will make it minikube start to be the only way to create a profile.